.github/workflows/linux-aarch64.yml

```diff
@@ -41,6 +41,10 @@ jobs:
           { config: "default", shard: 2, num_shards: 4, runner: "linux.arm64.2xlarge" },
           { config: "default", shard: 3, num_shards: 4, runner: "linux.arm64.2xlarge" },
           { config: "default", shard: 4, num_shards: 4, runner: "linux.arm64.2xlarge" },
+          { config: "default", shard: 1, num_shards: 3, runner: "linux.arm64.m7g.4xlarge" },
+          { config: "default", shard: 2, num_shards: 3, runner: "linux.arm64.m7g.4xlarge" },
+          { config: "default", shard: 3, num_shards: 3, runner: "linux.arm64.m7g.4xlarge" },
+          
```

Comment:
Nit: Extra new line (from your live meeting PR :p)

Reply:
Coding while other people are watching is stressful :P